src/Entities/Entity.cpp

```diff
@@ -899,6 +901,15 @@ void cEntity::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 			DetectCacti();
 		}
 
+		// Handle magma block damage
+		if (
+			IsMob() ||
```

Comment:
Does any mob receive magma damage? 
I'm not sure about blazes, ghost, zombie pigman...

Reply:
Thanks! I'll make sure to run the stylecheck before making pull requests. I just checked in SMP and you're right fireproof mobs don't take damage from magma blocks, I'll make new a pull request where fireproof mobs don't take damage from the magma blocks.

Reply:
And you don't have to comment you PR when you commit 😄 You can re-request a review in the top right corner. And because we're a small community PR review might take some time sometimes - so don't worry

Reply:
Okay thanks :) I'm new to this stuff